calscrape.py

```diff
@@ -5,9 +5,14 @@
 This is the main script for scraping and returning calendar data.
 """
 
+import json
+
 from modules.spatula import Spatula
 from modules.calparse import ParsedCal
-import json
+
+VERSION = "1.0"
```

Comment:
Bumpversion is a convenient utility for semantic versioning.
https://github.com/peritus/bumpversion
https://semver.org/

Reply:
Neat -- I'll check it out! Maybe a silly question, but what's the benefit of using a tool like this over simply modifying the version string in the code? Is it just a matter of doing it more programmatically?

Reply:
Yeah, exactly. To me the ideal is to have the version string populated by the git tags. I haven't tried this tool yet:
https://github.com/pypa/setuptools_scm